docs/keys/variable-self.md

```diff
@@ -0,0 +1,87 @@
+@typedef {String} can-stache/keys/variable/self *self
+@parent can-stache/keys/variable
```

Comment:
I don't think this should show up in the sidebar, or it should be under `can-stache/keys`